tests/test_pybats.py

```diff
@@ -24,6 +20,10 @@
 import spacepy.pybats.ram as ram
 import spacepy.pybats.gitm as gitm
 
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
```

Comment:
I like moving the mpl imports after the standard lib per PEP8. Putting them between standard lib and pybats, instead of after pybats, would 1. probably be more compliant with PEP 8 ("local" imports last) and 2. ensure that we still get the Agg backend even if pybats imports pyplot.

Reply:
Totally agree with the reasoning here; change reverted.

The reason I moved it is because PEP8-compliant linters really don't like the call to `matplotlib.use()` followed by more imports. This is not a problem with current versions of MPL, as `plt.use` can be called whenever; but as we're supporting older MPL versions, I didn't want to remove/move that line yet. So I compromised to try to make my linter happy.